src/components/CompareResults/RevisionRowExpandable.tsx

```diff
@@ -60,14 +60,18 @@ function RevisionRowExpandable(props: RevisionRowExpandableProps) {
       ((newMedian - baseMedian) / baseMedian) * 100,
     );
   }
+  const kde_new_x = (result as MannWhitneyResultsItem)?.kde_new?.kde_x ?? [];
+  const kde_base_x = (result as MannWhitneyResultsItem)?.kde_base?.kde_x ?? [];
 
   const baseValues =
-    baseRunsReplicates && baseRunsReplicates.length
+    baseRunsReplicates && (baseRunsReplicates ?? [])?.length
```

Comment:
It looks like we're doing two undefined/null checks here or are you checking for something else in the second part of the `&&` expression?